### PR TITLE
Bug/go to reference

### PR DESCRIPTION
**Pull request by FarisElghlan**
Fixed the go to base option. It now correctly goes to the specified base of the reference genome.

### CODE REVIEW

**Comment by cbilstra** [comment]
Apart from the fact that travis fails, the code looks good. You use some very short variable names like 'kv' but it is only inside some simple methods, so no need to adjust this for me. I scanned through all code, but I have not checked the code itself because the list of files changed was quite big.

**Comment by justinvdk** [comment]
merged #223.
@FarisElghlan could you rebase this against the (new) `dev` branch to filter out the changes that #223 brought with it?

**Comment by FarisElghlan** [comment]
Rebased.

File: PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/SearchPaneController.java [hunk 1]
```diff
@@ -225,7 +224,7 @@ private void initializeGoTo() {
     goToButton.setOnAction(actionEvent -> {
       String fieldText = goToField.getText();
       if (fieldText.matches("\\d+")) {
-        graphPaneController.centreOnLevel(Integer.valueOf(fieldText));
+        graphPaneController.centerOnLevel(Integer.valueOf(fieldText));
```
Comment (justinvdk) [inline PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/SearchPaneController.java:227]: Keep UK spelling please.

Comment (FarisElghlan) [inline PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/SearchPaneController.java:227]: Isn't US spelling a lot more common for programming?

**Pull request merged**